pkg/controller/s2ibinary/s2ibinary_controller.go

```diff
@@ -0,0 +1,256 @@
+package s2ibinary
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/s3"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/workqueue"
+	"k8s.io/klog"
+	"k8s.io/kubernetes/pkg/util/metrics"
+	"kubesphere.io/kubesphere/pkg/simple/client/s2is3"
+	"kubesphere.io/kubesphere/pkg/utils/sliceutil"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"time"
+
+	devopsv1alpha1 "kubesphere.io/kubesphere/pkg/apis/devops/v1alpha1"
+	devopsclient "kubesphere.io/kubesphere/pkg/client/clientset/versioned"
+	devopsinformers "kubesphere.io/kubesphere/pkg/client/informers/externalversions/devops/v1alpha1"
+	devopslisters "kubesphere.io/kubesphere/pkg/client/listers/devops/v1alpha1"
+)
+
+var log = logf.Log.WithName("s2ibinary-controller")
```

Comment:
replace with https://k8s.io/klog, since klog have log levels. we are going to use klog across whole project